index.md

```diff
@@ -9,24 +10,32 @@ toc: false
 
 ## Overview 
 
-GraphWalker is a [Model-Based testing](https://en.wikipedia.org/wiki/Model-based_testing) tool. It reads models in the shape of [directed graphs](https://en.wikipedia.org/wiki/Directed_graph), and generate [test] paths from these graphs.
+GraphWalker is a [Model-Based testing](https://en.wikipedia.org/wiki/Model-based_testing) tool. 
```

Comment:
I have transferred some sentences on new lines due to some lines are very long and it is not readable in IDE.
It requires horizontal scrolling and anybody who read it may lost focus on which line he was.